bin/www

```diff
@@ -0,0 +1,9 @@
+const http = require('http');
+const app = require('../app');
```

Comment:
Is it possible you try to remove this bin folder and see how you can start your server with just your express file